reflex/components/next/image.py

```diff
@@ -80,6 +83,27 @@ def create(
         Returns:
             _type_: _description_
         """
+        if "width" not in props or "height" not in props:
+            # props.pop("width", None)
+            # props.pop("height", None)
+            if "width" in props:
+                props["width"] = f"{props['width']}px"
+            if "height" in props:
+                props["height"] = f"{props['height']}px"
+                # return Box.create(
```

Comment:
What's the issue in using a Box?

Reply:
We've have to know both width and height if we want the box to constrain an image set to 100%, it won't work.